packages/spacecat-shared-utils/src/index.js

```diff
@@ -38,6 +38,8 @@ export {
 export { sqsWrapper } from './sqs.js';
 export { sqsEventAdapter } from './sqs.js';
 
+export { logWrapper } from './log-wrapper.js';
```

Comment:
Is `jobLogWrapper` too verbose? This wrapper is pretty specific to event-driven function invocation involving messages that include a job ID.

Reply:
Addressed in the reply to comment above ^^^^